docs/1-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -42,12 +42,24 @@ spec:
 apiVersion: v1
 kind: query
 spec:
-  name: Get authorized keys
+  name: Get authorized keys for Local Accounts
   platforms: macOS, Linux
   description: List authorized_keys for each user on the system.
   query: SELECT * FROM users CROSS JOIN authorized_keys USING (uid);
   purpose: Informational
   remediation: N/A
+  contributors: anelshaer
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get authorized keys for Domain Joined Accounts
+  platforms: macOS, Linux
+  description: List authorized_keys for each user on the system.
+  query: SELECT *  FROM users CROSS JOIN  authorized_keys USING(uid) WHERE  username IN (SELECT distinct(username) FROM last);
```

Comment:
OSquery have lots of capabilities that all comes back to the use of it. 

i use these queries to investigate a server local accounts for example, when you get notified by an admin/user about suspicious activity and you run these queries, then you can validate that with the admin/user is that something you know about or you didnot know about that change in the system. that's also applies to the Domain accounts when those servers are joined in domain controller env and there is no local account but root.

Reply:
> there is no local account but root.

Does this query handle that situation? It looks like there would still have to be a local user account in order to generate a row in this query, since the `last` table is only used as a filter in the `WHERE` clause.

I'm not opposed to adding the query, I just want to make sure it's working as intended.

Reply:
I did not know that additional user accounts would be available when `username` is specified like that! I need to get a Domain set up to do some experimenting with.